src/health_checker.cc

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+#include "health_checker.h"
+
+#include <boost/filesystem.hpp>
+#include <boost/algorithm/string/join.hpp>
+#include <vector>
+#include <fstream>
+
+namespace {
```

Comment:
Just realized that this should have been within `namespace google`, but it's a moot point now.